docs/getting-started.md

```diff
@@ -0,0 +1,38 @@
+# Getting started
+
+ 1. [Download the latest release version][1] and extract the repository using `tar`:
+
+    `tar -xzvf /path/to/github-backup-utils-vMAJOR.MINOR.PATCH.tar.gz`
```

Comment:
While it is mentioned in the requirements section, additional clarity around which version of the backup utilities to use if you are running GHE 2.10 or earlier would be valuable here, in particular if cloning the repository using `git`.